src/lib/routing-tables.js

```diff
@@ -56,7 +56,7 @@ class RoutingTables {
    * @param {Route|RouteData} _route from ledger B→C
    * @returns {Boolean} whether or not a new route was added
    */
-  addRoute (_route) {
+  addRoute (_route, prefix) {
```

Comment:
Now that I think about it, would `targetPrefix` be a better name? I think it would be helpful for differentiating between that and `destinationLedger`.